### ISSUE TITLE
[feature request] new chat message sound

### ISSUE THREAD

**Issue by RalphORama** [budak7273/NogsChat#1]
The option to enable a "ping" sound when another player in the server sends a chat message would be nice!

**Comment by budak7273** [comment]
I agree; I'll see if I can get this into the next version. However, the Update 8 release has been delayed by changes the base game has made to the chat window, so I'm not sure when that will come out.
